pippo-session-parent/pippo-session-xmemcached/src/main/java/ro/pippo/session/xmemcached/XmemcachedSessionDataStorage.java

```diff
@@ -75,14 +75,14 @@ public void save(SessionData sessionData) {
 
     @Override
     public SessionData get(String sessionId) {
```

Comment:
A suggestion:

```java
try {
    return this.sessions.get(sessionId);
} catch (TimeoutException | InterruptedException | MemcachedException ex) {
    log.error("An error occurred when get SessionData.", ex);
    return null;
}
```

without to declare the `sessionData` variable

I prefer to use few code lines to express an idea but without to sacrifice the readability.